db/kafka/docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       - all
       - kafka
       - kowl
+      - service
```

Comment:
Resolve profile error when running the service with `make up n=indexing p=service`